src/components/FormaUnosReg.jsx

```diff
@@ -0,0 +1,124 @@
+import { useState } from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
+function FormaUnosReg(props) {
+    const [formaPodaciReg, postaviPodatke] = useState({
+        username: "",
+        mail: "",
+        lozinka: ""
+      });
+    const [ispravanUnosUsername, postaviIspravanUnosUsername] = useState(true);
+    const [ispravanUnosMail, postaviIspravanUnosMail] = useState(true);
+    const [ispravanUnosLozinka, postaviIspravanUnosLozinka] = useState(true);
+    
+    const handleRegistracija = async (event) => {
+      event.preventDefault(); 
+
+      if (ispravanUnosUsername && ispravanUnosMail && ispravanUnosLozinka) {
+        registerUser(formaPodaciReg);
+      }  
+    };
+    function promjenaUlaza(event) {
+      const { name, value } = event.target;
+      if ( name === "username"  && /\d/.test(value)) {
+        postaviIspravanUnosUsername(false) 
+      }
+      else{
+        postaviIspravanUnosUsername(true);
+        postaviPodatke({ ...formaPodaciReg, [name]: value });
+      }
+      if ( name === "mail"  && !value.includes("@")) {
+        postaviIspravanUnosMail(false) 
+      }
+      else{
+        postaviIspravanUnosMail(true);
+        postaviPodatke({ ...formaPodaciReg, [name]: value });
+      }
+      if ( name === "lozinka" && value.length<7){
+        postaviIspravanUnosLozinka(false)
+      }
+      else{
+        postaviIspravanUnosLozinka(true);
+        postaviPodatke({ ...formaPodaciReg, [name]: value })
+      }
+      console.log(formaPodaciReg)
+    }
+    const registerUser = async (userData) => {
+      try {
+        const response = await axios.post('http://localhost:3000', {
+          headers: {
+            'Content-Type': 'application/json'
```

Comment:
cudno mi je da odi tribas naglasavat json, jel moze bez toga, ako ne moze vjv nam na BE il FE fali neki dependency

Reply:
moze al mi je ostalo jer san sa fetchon radia prije